firefox_ui_tests/functional/locationbar/test_access_locationbar.py

```diff
@@ -0,0 +1,59 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from firefox_ui_harness.decorators import skip_under_xvfb
+from firefox_ui_harness.testcase import FirefoxTestCase
+
+
+class TestAccessLocationBar(FirefoxTestCase):
+
+    def setUp(self):
+        FirefoxTestCase.setUp(self)
+
+        # Clear complete history so there's no interference from previous entries.
+        self.places.remove_all_history()
+
+        self.test_urls = [
+            'layout/mozilla_projects.html',
+            'layout/mozilla.html',
+            'layout/mozilla_mission.html'
+        ]
+        self.test_urls = [self.marionette.absolute_url(t)
+                          for t in self.test_urls]
+
+        self.locationbar = self.browser.navbar.locationbar
+        self.autocomplete_results = self.locationbar.autocomplete_results
+        self.urlbar = self.locationbar.urlbar
+
+    @skip_under_xvfb
```

Comment:
Do this skip at the class level so that we can avoid issue with tearDown and the autocomplete popup.